src/findlib/Makefile

```diff
@@ -123,7 +123,7 @@ clean:
 install: all
 	$(INSTALLDIR) "$(DESTDIR)$(prefix)$(OCAML_SITELIB)/$(NAME)"
 	$(INSTALLDIR) "$(DESTDIR)$(prefix)$(OCAMLFIND_BIN)"
-	$(INSTALLDIR) "$(DESTDIR)$(prefix)$(OCAML_CORE_STDLIB)"
+	test $(INSTALL_TOPFIND) -eq 0 || $(INSTALLDIR) "$(DESTDIR)$(prefix)$(OCAML_CORE_STDLIB)"
```

Comment:
It might be a different bug that can still occure for people who install ocamlfind without opam.

With and without coreutils, `install -d` will set the permissions to `755`. So if the user wanted to keep the permission of their installation directory the same, installing ocamlfind will change it without notice.

Reply:
I think that's worth opening in a separate issue. I'm not sure whether it should be `install -d`, `test -d || install -d`, or whether `mkdir -p` will do 🤷‍♂️